lib/did/document.ex

```diff
@@ -121,6 +121,10 @@ defmodule DID.Document do
     end
   end
 
+  def to_rdf(document) do
+    Grax.to_rdf!(document) |> RDF.Dataset.new()
```

Comment:
I wouldn't convert it to an RDF Dataset, per se. Generally, it's easier to deal with a single graph. I would consider the dataset conversion an implementation detail of the test this was extracted from.

Reply:
Makes sense. I would say let's work with datasets as little as possible then and instead convert the expected value with `RDF.Dataset.default_graph()` so we usually compare on a graph level.

Reply:
@marcelotto please check my latest commit